tensorboard/plugins/scalar/tf_scalar_dashboard/tf-scalar-dashboard.html

```diff
@@ -245,6 +245,10 @@ <h3>No scalar data was found.</h3>
         return _smoothingWeight > 0;
       },
 
+      _shouldOpen(index) {
```

Comment:
Should we also update `tf-pr-curve-dashboard.html` and `tf-text-dashboard.html`?

Reply:
Done